jira-feature-collector/src/main/java/com/capitalone/dashboard/datafactory/jira/JiraDataFactoryImpl.java

```diff
@@ -254,8 +255,20 @@ public String setQuery(String query) {
 				}
 			} catch (Exception e) {
```

Comment:
@amitmawkin , **Correction**:  I have made the exception update to my PR.  Please review my latest commit change:  https://github.com/capitalone/Hygieia/pull/606/commits/0eb9af2a63f0c580d5d2fa793e7a5e15d54d3ee1 .  Feel free to move the discussion over to that PR.